apps/mochi-web/components/TransactionTable/types.ts

```diff
@@ -0,0 +1,35 @@
+import { PaginationProps, TableProps } from '@mochi-ui/core'
+
+export interface Tx {
+  code: string
+  where: {
+    text: string
+    avatar: string | ((...props: any[]) => JSX.Element)
+  }
+  from: {
+    address: string
+    avatar: string
+    platform?: string
+    platformIcon?: string
+  }
+  to: {
+    address: string
+    avatar: string
+    platform?: null | string
+    platformIcon?: string
+  }
+  token: {
+    icon: string
+    symbol: string
+  }
+  action: string
+  amount: string
+  date: string
+  isSuccess?: boolean
+}
+
+export type TransactionTableProps = Omit<TableProps<Tx>, 'columns'> & {
```

Comment:
Can you provide a prop to hide column(s)? I believe there may be where the table is intended for a narrow scope, such as on the Tip page, and therefore the Type column would be unnecessary in such instances.